packages/playwright-core/src/client/fetch.ts

```diff
@@ -80,7 +77,7 @@ export class APIRequest implements api.APIRequest {
     this._contexts.add(context);
     context._request = this;
     context._tracing._tracesDir = tracesDir;
-    await this._onDidCreateContext?.(context);
+    await this._playwright._instrumentation.onDidCreateRequestContext(context);
```

Comment:
To be similar to other places:
```suggestion
    await this._instrumentation.onDidCreateRequestContext(context);
```

Reply:
This unfortunately won't compile. You probably mean context._instrumentation. But it won't be similar.